modules/kubernetes-addons/aws-load-balancer-controller/locals.tf

```diff
@@ -39,11 +39,16 @@ locals {
     serviceAccountName = local.service_account_name
   }
 
-  irsa_config = {
-    kubernetes_namespace              = "kube-system"
+  default_irsa_config = {
+    kubernetes_namespace              = local.helm_config["namespace"]
     kubernetes_service_account        = local.service_account_name
-    create_kubernetes_namespace       = false
+    create_kubernetes_namespace       = var.kubernetes_namespace != "kube-system" ? true : false
```

Comment:
replace this with 
```hcl
create_kubernetes_namespace  = try(local.helm_config["create_namespace"], true)
```

Reply:
replace this line in  **IRSA module**  https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/91dc3651d33049029f7b4188b515ab16a1b11a24/modules/irsa/main.tf#L2

with the below

```hcl
count = var.create_kubernetes_namespace && var.kubernetes_namespace != "kube-system" ? 1 : 0
```